resources/puppetlabs/puppetdb/pql/pql-grammar-experimental.ebnf

```diff
@@ -30,7 +28,7 @@ extract = <lbracket>, [<whitespace>], [extractfields], [<whitespace>], <rbracket
 <extractfields> = fieldlist;
 
 (* Filtering *)
-<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], [modifier], [<whitespace>], <rbrace>;
+<where> = <lbrace>, [<whitespace>], [expression], [<whitespace>], {modifier | pagingclause, [<whitespace>]}, [<whitespace>], <rbrace>;
```

Comment:
I went back and forth on this, but settled on splitting them out because they end up in different spots in the final query. Group by is always inside an extract, but the paging clauses go outside the extract and inside the surrounding from. I'd be fine with making group by its own thing or whatever we settle on.

I'm not sure what you mean by two modifiers/pagination parameters. Two paging parameters will work, but I think that this query

```
reports[status, count()]{group by status limit 1]
```

is unfortunately ambiguous under the language. Erroring for now would probably be best in my opinion, happy to put that in.